src/js/changeToUnpressed.js

```diff
@@ -0,0 +1,18 @@
+function changeToUnpressed(button, forceShiftToggle = false) {
+    const shiftLeft = document.querySelector('.ShiftLeft');
+    const shiftRight = document.querySelector('.ShiftRight');
+
+    if (button.classList.contains('ShiftLeft') || button.classList.contains('ShiftRight') || forceShiftToggle) {
+        shiftLeft.classList.toggle('keyboard__button_activate');
+        shiftRight.classList.toggle('keyboard__button_activate');
+    } else if (button.classList.contains('CapsLock')) {
+        button.classList.toggle('keyboard__button_activate');
+    } else {
+        setTimeout(() => {
+            button.classList.remove('active');
+        }, 250);
+    }
+    return false;
+}
+
+export default changeToUnpressed;
```

Comment:
I guess, you can combine this function with changeToPressed

Reply:
Done